tests/Transport/BaseTestCase.php

```diff
@@ -49,18 +49,20 @@ public function testResponseByteLimit() {
 		$limit    = 104;
 		$options  = [
 			'max_bytes' => $limit,
+			'hooks'     => $this->getMaxBytesAssertionHooks(),
 		];
-		$response = Requests::get(httpbin('/bytes/325'), [], $this->getOptions($options));
+		$response = Requests::get(httpbin('/bytes/1000000'), [], $this->getOptions($options));
```

Comment:
I used high numbers because curl might decide to use bigger chunks, and I wanted to ensure that we can reliably say that not only a single chunk going past the limit is truncated, but also that all subsequent chunks have been suppressed.